app/models/generic_event/person_move_death_in_custody.rb

```diff
@@ -0,0 +1,14 @@
+class GenericEvent
```

Comment:
This and the next few classes are identical apart from naming, might be a nice use case for inheritance to avoid repeating the same stuff over and over?

Reply:
I was thinking that, yeah.

Reply:
Will have a play with it

Reply:
Done